test/OptionsCompounder.t.sol

```diff
@@ -96,13 +112,17 @@ contract OptionsTokenTest is Test {
         paymentToken = IERC20(WETH);
         underlyingToken = IERC20(OATH);
         address[] memory strategists = new address[](1);
-        //address[] memory multisigRoles = new address[](3);
+        address[] memory multisigRoles = new address[](3);
+        address[] memory keepers = new address[](1);
         strategists[0] = strategist;
-        // multisigRoles[0] = management1;
-        // multisigRoles[1] = management2;
-        // multisigRoles[2] = management3;
-        address[] memory strategies = new address[](1);
-        strategies[0] = strategy;
+        multisigRoles[0] = management1;
+        multisigRoles[1] = management2;
+        multisigRoles[2] = management3;
+        keepers[0] = keeper;
+
+        // Question: Doesn't matter in this test but what are the reasonable values ??
```

Comment:
for purposes of this test 0 is good. 0 to 1e18 is the actual possible range.  It is representing a ratio of 0-1